components/constellation/constellation.rs

```diff
@@ -1346,8 +1338,12 @@ where
             FromCompositorMsg::TraverseHistory(webview_id, direction) => {
                 self.handle_traverse_history_msg(webview_id, direction);
             },
-            FromCompositorMsg::WindowSize(webview_id, new_size, size_type) => {
-                self.handle_window_size_msg(webview_id, new_size, size_type);
+            FromCompositorMsg::ChangeViewportDetails(
+                webview_id,
+                new_viewport_details,
+                size_type,
+            ) => {
+                self.handle_window_size_msg(webview_id, new_viewport_details, size_type);
```

Comment:
Should the method also be renamed to match the enum variant?

Reply:
Yep, this is done now in the latest version.